### PR TITLE
Added support for new assets

### PR DESCRIPTION
**Pull request by vnavascues**

### CODE REVIEW

File: rotkehlchen/assets/converters.py [hunk 1]
```diff
@@ -321,9 +321,6 @@
     # YinCoin. No data found except from here:
     # https://coinmarketcap.com/currencies/yincoin/
     'YIN',
-    # LINK, BEAR/BULL. No data found yet but should probably revisit
-    'LINKBULL',
```
Comment (vnavascues) [inline rotkehlchen/assets/converters.py:325]: We do support them now, available in coingecko and cryptocompare.

File: rotkehlchen/constants/assets.py [hunk 2]
```diff
@@ -70,6 +70,5 @@
 A_GRAIN = EthereumToken('GRAIN')
 A_COMBO = EthereumToken('COMBO')
 A_LDO = EthereumToken('LDO')
-A_GUSD = EthereumToken('GUSD')
```
Comment (vnavascues) [inline rotkehlchen/constants/assets.py:73]: removed

File: rotkehlchen/data/all_assets.json [hunk 3]
```diff
@@ -15008,5 +15008,96 @@
         "started": 1608242396,
         "symbol": "LDO",
         "type": "ethereum token"
+    },
+    "BETH": {
```
Comment (LefterisJP) [inline rotkehlchen/data/all_assets.json:15012]: Remove this and just add a mapping in the binance exchange to ETH2. Beacon ETH is essentially ETH2, just from inside the binance exchange.

File: docs/changelog.rst [hunk 4]
```diff
@@ -43,6 +43,17 @@ Changelog
   - `Invictus Hyperoin Fund (IHF) <https://www.coingecko.com/en/coins/invictus-hyperion-fund>`__
   - `Flow - Dapper labs (FLOW) <https://www.coingecko.com/en/coins/flow>`__
   - `Lido DAO (LDO) <https://www.coingecko.com/en/coins/lido-dao>`__
+  - `Binance Beacon ETH (BETH) <https://www.cryptocompare.com/coins/beth/overview>`__
+  - `DeXe (DEXE) <https://www.coingecko.com/en/coins/dexe>`__
+  - `Trust Wallet Token (TWT) <https://www.coingecko.com/en/coins/trust-wallet-token>`__
+  - `Meaconcash (MCH) <https://www.coingecko.com/en/coins/meconcash>`__
+  - Singapore Dollar (SGD)
```
Comment (LefterisJP) [inline docs/changelog.rst:50]: Remove this from this list and add a new full changelog entry. This is how we have done it in the past for FIAT.

**Review by LefterisJP — COMMENTED** [review]
Some minor comments.

Also please make sure to run

`python pytestgeventwrapper.py -xs  rotkehlchen/tests/unit/test_assets.py::test_coingecko_identifiers_are_reachable`

and make sure no warnings are there. Also in the rest of test_assets.py see if there is any warnings we should handle.

File: rotkehlchen/assets/asset.py [hunk 5]
```diff
@@ -153,7 +153,7 @@
     # Solana is SOL-2 in rotki
     'SOL-2': 'SOL',
     # BETH is the eth staked in beacon chain
-    'BETH': 'ETH2',
```
Comment (LefterisJP) [inline rotkehlchen/assets/asset.py:156]: lol I had it the other way around before!

**Review by LefterisJP — APPROVED** [review]
LGTM

**Comment by LefterisJP** [comment]
2 test failures only this time, both kraken asset pairs.

**Pull request merged**